src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = this[0];
+    let i = 1;
+
+    if ([...arguments].length > 1) {
```

Comment:
Is it necessary to spread arguments here ?

```suggestion
    if (arguments.length > 1) {
```